views/containers.html

```diff
@@ -1,7 +1,7 @@
 <div ng-if="!containers.length"><em>none</em></div>
 <dl class="dl-horizontal" ng-repeat="container in containers">
 <dt>Name</dt>
-<dd>{{container.name}}</dd>
```

Comment:
True, though hawtio-extension is just a directive.  If it hasn't been loaded, then the <span> is effectively a noop.